src/dataloaderinterface/static/dataloaderinterface/js/browse-sites.js

```diff
@@ -333,9 +333,7 @@ $(document).ready(function () {
 
                 markers[i].setVisible(visible);
 
-                if (!someVisible && visible) {
-                    someVisible = true;
-                }
+                someVisible = someVisible || (!someVisible && visible)
```

Comment:
thank you and thank you and thank you. if statements to declare a boolean value are nonsense.